cpp/include/core/MctsConfigurationBase.hpp

```diff
@@ -0,0 +1,10 @@
+#pragma once
+
+namespace core {
+
+struct  MctsConfigurationBase {
+  static constexpr float kOpeningLength = 0;
+  static constexpr bool kStoreStates = false;
+};
+
+} // namespace core
```

Comment:
Please get into the habit of including one blank newline at the end of each file. Most style-guides ask for this.